tests/phpunit/tests/rest-api/wpRestMenuItemsController.php

```diff
@@ -1053,4 +1053,26 @@ public function test_update_item_properly_handles_slashed_data() {
 		$new_data = $response->get_data();
 		$this->assertSame( $params['title'], $new_data['title']['raw'] );
 	}
+
+	/**
+	 * @ticket 61842
+	 * @covers ::create_item
+	 */
+	public function test_create_item_invalid_type_for_url() {
```

Comment:
This unit test doesn't cover the error described in the ticket.
Would you consider adding a unit test that attempts to submit an array for `url` instead of just a string?
It seems this unit test doesn't trigger the check you added to the custom `validate_callback` function.